libs/src/LSPlugin.core.ts

```diff
@@ -195,17 +209,26 @@ enum PluginLocalLoadStatus {
   UNLOADING = 'unloading',
   LOADED = 'loaded',
   UNLOADED = 'unload',
-  ERROR = 'error'
+  ERROR = 'error',
```

Comment:
Better avoid code style and whitespace changes.

Reply:
Hmm. We defined `.prettierrc.js`, but the files inside the libs don't follow it.

It looks like we need a git hook that will automatically unify the code before committing. I would like to open a new PR to handle this, with [husky](https://typicode.github.io/husky/#/) and [lint-staged](https://github.com/okonet/lint-staged). What do u think?

Reply:
husky did the ci stage thing. personally, I don't think it's very good practice.
Better and code style check to CI.
CC @xyhp915 @logseq-cldwalker

Reply:
The advantage is that developers don't care about any lint checks, and all possible formatting problems can be dealt with locally.

Reply:
I created #4760 to discuss something related to this.

Reply:
@g1eny0ung Current lint rules of libs TS files follow StandardJS https://standardjs.com/. There's a prettier package for those rules https://www.npmjs.com/package/prettier-standard. But for sure, will support commit hooks for lint task.